src/OfflineWorkflowsSample/OfflineWorkflowsSample/ViewModels/LoginViewModel.cs

```diff
@@ -127,7 +127,7 @@ private async Task<ArcGISPortal> AuthenticateAndCreatePortal()
                 var cred = await AuthenticationManager.Current.GetCredentialAsync(cri, true);
 
                 // Create the portal with authentication info
-                return await ArcGISPortal.CreateAsync(cred.ServiceUri, cred);
```

Comment:
second parameter should be `true` now to ensure portal will be signed into (a portal allowing anonymous users could load without a user set)